lib/pages/main/wallet_contents/add_wallet_connect/components/add_wallet_connect_mobile_view.dart

```diff
@@ -96,6 +96,25 @@ class _AddWalletConnectMobileViewState
                   ),
                 ),
               ),
+            // Text below the scan window
```

Comment:
@ahmed-tarek-salem  please move up in the screen the frame to scan and the instructions

Reply:
Fixed with b75a593, tested on large and small mobile screens.